archetype/package.json

```diff
@@ -1,27 +1,25 @@
 {
   "version": "0.0.0",
   "keywords": [
-    "trails",
-    "trailsjs"
+    "trails"
   ],
   "main": "index.js",
   "dependencies": {
-    "trailpack-repl": "^2",
-    "trailpack-router": "^2",
-    "trails": "^2",
-    "winston": "^2.3"
+    "trailpack-router": "next",
```

Comment:
this cause lot of trouble each time, better to have ^3

Reply:
Little up here :) I think it's important to prevent all people who use the generator to have crash